ee/LICENSE

```diff
@@ -0,0 +1,42 @@
+The PrairieLearn Enterprise Edition (EE) license (the “EE License”)
+Copyright (c) 2021-present PrairieLearn, Inc.
+
+With regard to the PrairieLearn Software:
+
+This software and associated documentation files (the "Software") may only be
+used in production, if you (and any entity that you represent) have agreed to,
+and are in compliance with, the terms of a contract, subscription, or other
+agreement governing the use of the Software, as agreed by you and PrairieLearn,
+Inc., and otherwise have a valid PrairieLearn Enterprise Edition subscription.
+Subject to the foregoing sentence, you are free to modify this Software and
+publish patches to the Software. You agree that PrairieLearn, Inc. and/or its
+licensors (as applicable) retain all right, title and interest in and to all
+such modifications and/or patches, and all such modifications and/or patches
+may only be used, copied, modified, displayed, distributed, or otherwise
+exploited with a valid PrairieLearn Enterprise Edition subscription.
+Notwithstanding the foregoing, you may copy and modify the Software for
+development and testing purposes, without requiring a subscription. You are not
+granted any other rights beyond what is expressly stated herein. Subject to the
+foregoing, it is forbidden to copy, merge, publish, distribute, sublicense,
+and/or sell the Software.
+
+This EE License applies only to the part of this Software that is not
+distributed as part of PrairieLearn Community Edition (CE). Any part of this
+Software distributed as part of PrairieLearn CE or is served client-side as an
+image, font, cascading stylesheet (CSS), file which produces or is compiled,
+arranged, augmented, or combined into client-side JavaScript, in whole or in
+part, is copyrighted under the GNU Affero General Public License. The
```

Comment:
I just realized that this conflicts with what's said in `/LICENSE` - should we just license everything under AGPL for consistency? It's not clear to me if there's a benefit to licensing any client-side JS specifically as MIT.

Reply:
Yeah, remove all the client-side special cases, I think.

Reply:
Discussed offline with Matt and we're going to leave the client-side clause in, but just change it to AGPL.